packages/firebase_remote_config/ios/firebase_remote_config.podspec

```diff
@@ -0,0 +1,23 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+Pod::Spec.new do |s|
```

Comment:
Update the values in this file to match the podspec of other plugins

Reply:
Done